oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java

```diff
@@ -339,6 +344,40 @@ public boolean updateAttributes() {
         return true;
     }
 
+    /**
+     * If some details couldn't be read from ProcPath.PID_STATUS try reading it from
+     * ProcPath.PID_STAT
+     *
+     * @param status
+     *            status map to fill.
+     * @param stat
+     *            string to read from.
+     */
+    private static void getMissingDetails(Map<String, String> status, String stat) {
+        if (status == null || stat == null) {
+            return;
+        }
+
+        int nameStart = stat.indexOf("(");
+        int nameEnd = stat.indexOf(")");
+        if (Util.isBlank(status.get("Name")) && nameStart < nameEnd) {
+            // remove leading and trailing parentheses
+            String statName = stat.substring(nameStart, nameEnd + 1);
+            statName = statName.substring(1, statName.length() - 1);
+            status.put("Name", statName);
+        }
+
+        // get the rest of the string from after the name
+        // (the name may contain whitespaces and destroy the index)
+        String statEnd = stat.substring(nameEnd + 2);
+        String[] statSplit = ParseUtil.whitespaces.split(statEnd);
```

Comment:
I think this is unnecessary. You only need the single character, so `stat.charAt(nameEnd + 2)` should return this.  You'll need to do a length check, of course.